testutil/car.go

```diff
@@ -181,3 +209,20 @@ func WriteUnixfsDAGTo(path string, into ipldformat.DAGService, chunksize int64,
 
 	return nd.Cid(), nil
 }
+
+func CidsInCar(t *testing.T, path string) []cid.Cid {
+	outReader, err := os.Open(path)
+	require.NoError(t, err)
+	blockReader, err := car.NewBlockReader(outReader)
+	require.NoError(t, err)
+	gotCids := []cid.Cid{}
+	for {
+		next, err := blockReader.Next()
+		if err != nil {
+			break
+		}
+		gotCids = append(gotCids, next.Cid())
+	}
+	require.NoError(t, err)
+	return gotCids
+}
```

Comment:
I think we can remove the `testing.T` from this function and simply return the error from here. It would make for a more uniform util func.